draft-ietf-bidding-and-auction-services.md

```diff
@@ -391,10 +391,11 @@ the {{framing}} header.
 This section describes how the client MAY form and serialize request messages
 in order to communicate with the Bidding and Auction services.
 
-This algorithm takes as input all of the `relevant interest groups`, and a config
-consisting of the `publisher`,
+This algorithm takes as input all of the `relevant interest groups`, a config
+consisting of the `publisher`, a map of from (origin, string) tuple to origin

```

Comment:
I would have thought we could calculate `ig pagg coordinators` from the `relevant interest groups`. I think we'd rather do that than have it as a separate argument.

Reply:
but it seems step 10.4 of https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#section-2.2.4 serialized everything

Reply:
Ah, I didn't realize we weren't explicitly serializing it. Then I guess we do need to pass it separately.